corehq/apps/cloudcare/touchforms_api.py

```diff
@@ -18,7 +18,6 @@ def get_session_data(self, device_id=CLOUDCARE_DEVICE_ID):
         """
         session_data = {
             'device_id': device_id,
-            'app_version': '2.0',
```

Comment:
This looks like [`APP_V2`](https://github.com/dimagi/commcare-hq/blob/f40fe7c8069192ff00403b7d1d3a6029a1e89c6c/corehq/apps/app_manager/const.py#L2) as in [`Application.application_version`](https://github.com/dimagi/commcare-hq/blob/f40fe7c8069192ff00403b7d1d3a6029a1e89c6c/corehq/apps/app_manager/models.py#L4191), which has not seen a new version in years. Is this PR changing the meaning of `session_data["app_version"]` to now be [`Application.version`](https://github.com/dimagi/commcare-hq/blob/f40fe7c8069192ff00403b7d1d3a6029a1e89c6c/corehq/apps/app_manager/models.py#L4112) (which is an integer, not a string)?

Does anything in formplayer depend on the old meaning and maybe needs to be updated?

Reply:
Yep this is changing the meaning. From the description "Currently nothing in formplayer makes use of that property which is why it is safe to change what this value represents", but I can make that more clear.

Reply:
I missed that in the PR description, so thanks for clarifying here. No need for further clarification.